Sources/Workspace/InitPackage.swift

```diff
@@ -611,6 +614,38 @@ public final class InitPackage {
         try writeTestFileStubs(testsPath: tests)
     }
 
+    private func writeLibraryDocumentation(_ path: AbsolutePath) throws {
+        let docc = path.appending("\(moduleName).docc")
+        guard fileSystem.exists(docc) == false else {
+            return
+        }
+        try makeDirectories(docc)
+
+        let content = ####"""
+        # ``\####(moduleName)``
+
+        <!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+
+        ## Overview
+
+        <!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+
+        ## Topics
+
+        ### <!--@START_MENU_TOKEN@-->Group<!--@END_MENU_TOKEN@-->
+
+        - <!--@START_MENU_TOKEN@-->``Symbol``<!--@END_MENU_TOKEN@-->
+        """####
```

Comment:
I think it would be better to not put anticipatory content in here, and it's especially important to not use Xcode-specific template content.

```suggestion
        let content = "# ``\(moduleName)``\n\n"
```